pwa/src/data/categories.tsx

```diff
@@ -26,10 +26,10 @@ import {
 } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
-export type TCategories = "interactie" | "proces" | "integratie" | "services" | "data";
+export type TCategories = "Interactie" | "Proces" | "Integratie" | "Service" | "Data";
```

Comment:
I've left some other comments regarding this. But I think a general comment about this might be helpful.

I'd opt to keep this type lowercase (so: `"interactie", "..."`) instead of refactoring it to uppercase.

Afterwards, when we need it in another format (e.g. `upperFirst`) we can do so in the implementation.

This way, we stay more consistent with the way we create types (basically, this type holds a few keys and keys are not uppercase usually).

Note: it might be that the gateway returns its values with uppercase (I'm not sure), but those values we could `_.toLower()` before matching them with this type.